### PR TITLE
Dmcandrew update ruamel.yaml

### PR DESCRIPTION
**Pull request by dcmcand**
## What does this implement/fix?

_Put a `x` in the boxes that apply_

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds a feature)
- [ ] Breaking change (fix or feature that would cause existing features not to work as expected)
- [ ] Documentation Update
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no API changes)
- [ ] Build related changes
- [ ] Other (please describe):

## Testing

- [ ] Did you test the pull request locally?
- [ ] Did you add new tests?

### CODE REVIEW

**Review by marcelovilla — APPROVED** [review]
Looks good!

My only suggestion is we update the Python version in the docs too: https://github.com/nebari-dev/nebari-docs/blob/b436c7e09f874665f9379ee2fff0c5e249ae5cd0/docs/docs/get-started/installing-nebari.md?plain=1#L22

File: .github/workflows/test_conda_build.yaml [hunk 1]
```diff
@@ -54,5 +54,4 @@ jobs:
           conda build nebari
       - name: Test nebari
         run: |
-          conda install --use-local nebari
```
Comment (marcelovilla) [inline .github/workflows/test_conda_build.yaml]: @dcmcand why did you remove the install? Isn't it needed for `nebari` to work?

Comment (dcmcand) [reply .github/workflows/test_conda_build.yaml]: It was installing the wrong version.
